src/client/sandbox/node/live-node-list/html-collection-wrapper.js

```diff
@@ -118,27 +98,58 @@ function updateHtmlCollectionElementGetters (wrapper, oldLength, currentLength)
     while (oldLength > currentLength)
         delete wrapper[--oldLength];
 
-    if (currentLength > shadowGettersCount - DEFAULT_SHADOW_GETTERS_SHIFT)
-        addShadowGetters(currentLength - (shadowGettersCount - DEFAULT_SHADOW_GETTERS_SHIFT));
+    if (currentLength > collectionProtoGettersCount - COLLECTION_PROTO_GETTERS_RESERVE)
```

Comment:
Extract `collectionProtoGettersCount - COLLECTION_PROTO_GETTERS_RESERVE` to variable.
I propose to call it as `prevCollectionProtoGettersCount`.

Reply:
What about `maxCollectionLength`?